samples/client/petstore/python/petstore_api/api_client.py

```diff
@@ -116,7 +113,7 @@ def __call_api(self, resource_path, method,
                                                     collection_formats)
             for k, v in path_params:
                 resource_path = resource_path.replace(
-                    '{%s}' % k, quote(str(v), safe=''))  # no safe chars, encode everything
+                    '{%s}' % k, quote(str(v)))
```

Comment:
This change should be reverted to the upstream version in any case since it is irrelevant to this PR (and the former variant with `safe=''` is the correct one). Hopefully, the mentioned commits will land into 2.3.0 branch soon.